apiserver/allfacades.go

```diff
@@ -149,6 +149,9 @@ func AllFacades() *facade.Registry {
 	reg("Backups", 2, backups.NewFacadeV2)
 	reg("Block", 2, block.NewAPI)
 	reg("Bundle", 1, bundle.NewFacadeV1)
+	if featureflag.Enabled(feature.DeveloperMode) {
```

Comment:
You do not need feature flag here. 
Also, you are upping the facade version and in this collection only one instance of it can exist.